pkg/controller/constants/errors.go

```diff
@@ -0,0 +1,12 @@
+package constants
+
+const (
+	// ErrTrackPCUsage indicates tracking errors
+	ErrTrackPCUsage = "cannot track ProviderConfig usage"
+	// ErrGetPC indicates inaccessible ProviderConfig
+	ErrGetPC = "cannot get ProviderConfig"
+	// ErrNoSecretRef indicates inexistent SecretRef
+	ErrNoSecretRef = "ProviderConfig does not reference a credentials Secret"
+	// ErrGetSecret indicates inaccessible secret
+	ErrGetSecret = "cannot get credentials Secret"
+)
```

Comment:
This would be a new convention for Crossplane; we typically expose error strings as private constants in the same package they're used to make it easy to compare errors in tests. What is the advantage of creating a dedicated package for them?

Reply:
The idea was to share the common error strings between the reconcilers instead of duplicating them. But I don't have a strong preference on it.

Reply:
I slightly prefer to maintain the status quo here, but I also don't feel strongly. :)

Reply:
I changed it back 👍